ecf/scripts/enkfgdas/analysis/create/jenkfgdas_diag.ecf

```diff
@@ -0,0 +1,51 @@
+#PBS -S /bin/bash
```

Comment:
These EnKF jobs are new and we agreed to add them in a followup PR to avoid further bloating this current PR.